packaging/ansible-runner-service-project/project/roles/hc-gluster-cgroups/tasks/main.yml

```diff
@@ -1,26 +1,27 @@
 ---
 - name: Create systemd overrides directory for glusterd
-  file:
+  ansible.builtin.file:
     path: /etc/systemd/system/glusterd.service.d/
     state: directory
+    mode: '755'
 
 - name: Create glusterd cgroups CPU configuration file
-  copy:
+  ansible.builtin.copy:
     src: 99-cpu.conf
     dest: /etc/systemd/system/glusterd.service.d/99-cpu.conf
     mode: preserve
   notify:
     - Restart glusterd
 
 - name: Set CPU quota
-  template:
+  ansible.builtin.template:
     src: glusterfs.slice.j2
     dest: /etc/systemd/system/glusterfs.slice
+    mode: '644'
   vars:
-    gluster_cgroup_cpu_quota : "{{ [(ansible_processor_vcpus/3)|int,1]|max * 100 }}"
+    gluster_cgroup_cpu_quota: "{{ [(ansible_processor_vcpus / 3) | int, 1] | max * 100 }}"
```

Comment:
Shouldn't there be a space also after the slash?

Reply:
about the titles- it's because the title is long, looks like it was split automatically
space- yes, I fixed it in a commit I posted today: https://github.com/oVirt/ovirt-engine/pull/684/commits/0575c14191691ac26de0eef3f7aaa791b2e1503a

Reply:
> about the titles- it's because the title is long, looks like it was split automatically 

Please don't respond to comments in unrelated places, otherwise it's hard to track discussions.

> space- yes, I fixed it in a commit I posted today: [0575c14](https://github.com/oVirt/ovirt-engine/commit/0575c14191691ac26de0eef3f7aaa791b2e1503a)

Great, OK.